bench/run_workload_bench.sh

```
@@ -0,0 +1,574 @@
+NOISE_MARGIN="${NOISE_MARGIN:-0.08}"
+LK_BIN="/Users/lk/proj/lk/target/release/lk"
+LUA_BIN="lua"
+RUN_AOT="${RUN_AOT:-0}"
+AOT_ENABLED=0
+PROFILE_WORKLOADS="${PROFILE_WORKLOADS:-0}"
+BENCH_TIMEOUT="${BENCH_TIMEOUT:-30}"
+BENCH_PROGRESS="${BENCH_PROGRESS:-1}"
+LK_PREWARM_TIMEOUT="${LK_PREWARM_TIMEOUT:-120}"
+
+TMPDIR=$(mktemp -d)
+trap "rm -rf $TMPDIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)